UnityProject/Assets/Scripts/Items/Weapons/Projectiles/Behaviours/ProjectileDamageLivingHealth.cs

```diff
@@ -30,7 +31,7 @@ private bool TryDamage(MatrixManager.CustomPhysicsHit hit)
 			var coll = hit.CollisionHit.GameObject;
 			if (coll == null) return false;
 
-			var livingHealth = coll.GetComponent<LivingHealthBehaviour>();
+			var livingHealth = coll.GetComponent<LivingHealthMasterBase>();
```

Comment:
Fixed it in my other PR, as this would stop mobs from being damaged

Reply:
Though that would stop players from being damaged?

Reply:
oh xD misreading